js/contributor-cards/index.js

```diff
@@ -5,6 +5,10 @@ import contributorList from '../../data/contributors/jakarta-ee-9.json';
 
 const cardList = document.querySelector('.card-list.contributor-list');
 
+if (window.NodeList && !NodeList.prototype.forEach) {
+  NodeList.prototype.forEach = Array.prototype.forEach;
+}
+
```

Comment:
@autumnfound Is this snippet also not allowed? this is from https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach 
If so, I will remove it and make this PR fix IE images first. And find a way to add polyfill later.

Reply:
removed